index.html

```diff
@@ -138,7 +153,7 @@ <h6 class="thin text-muted">Cloud Monitoring and Deployment Tool written in Type
 				<h2>Join SIESGSTarena |</h2>
 				<br>
 				<h6 class="heading">Interested in solving challenging problems or building products? SIESGSTarena is the right community for you!</h6>
-				<h6 class="heading">For any queries, send an email to <em>codechef [at] siesgst [dot] ac [dot] in</em></h6>
```

Comment:
It was previously written in this way so that crawlers won't find our email and spam us right? Are we sure to change it to normal?

Reply:
I am sure about that. I will revert it back to normal.